src/metabase/models/interface.clj

```diff
@@ -260,24 +254,62 @@
      (Database)                       ; return a seq of *all* Databases (as instances of `DatabaseInstance`)
      (Database 1)                     ; return Database 1"
   {:arglist      '([entity table-name] [entity docstr? table-name & korma-forms])
-   :style/indent 1}
+   :style/indent 2}
   [entity & args]
   (let [[docstr [table-name & korma-forms]] (u/optional string? args)
         instance                            (symbol (str entity "Instance"))
         map->instance                       (symbol (str "map->" instance))]
     `(do
        (defrecord ~instance []
          clojure.lang.IFn
-         (~'invoke [this#]     (invoke-entity-or-instance this#))
-         (~'invoke [this# id#] (invoke-entity-or-instance this# id#)))
+         (~'invoke [this#]
```

Comment:
[unfortunately since `clojure.lang.IFn` is a Java interface](https://github.com/clojure/clojure/blob/master/src/jvm/clojure/lang/IFn.java) (not a Clojure protocol) we have to do it that way. Luckily that's an internal implementation detail so no one besides me ever looks at it.